CoreAnalyticView/src/au/gov/asd/tac/constellation/views/analyticview/results/AnalyticResult.java

```diff
@@ -78,6 +79,35 @@ public final int size() {
     }
 
     public final void sort() {
+        result = sortMapByValuesWithDuplicates(result);
+    }
+
+    private LinkedHashMap sortMapByValuesWithDuplicates(Map<IdentificationData, D> passedMap) {
```

Comment:
I'm wondering whether of all this can be a void function which is simply applied directly to the `result` map? That would also allow `result` to remain final. That would probably mean the following changes to your function to allow that to happen:
- replace calls of `passedMap` with `result`
- `sortedMap` declaration just becomes `result.clear()`
- adding to `sortedMap` would then instead add to `result`
- your key set map should then become an entry set map to allow the comparison to occur